packages/adminrouter/extra/src/test-harness/tests/test_generic.config.yml

```diff
@@ -706,19 +710,11 @@ endpoint_tests:
           - expected: /v2/reflect/me
             sent: /marathon/v2/reflect/me
           - expected: /
-            sent: /marathon
-          - expected: /
             sent: /marathon/
-      is_location_header_rewritten:
+      is_endpoint_redirecting_properly:
         enabled: true
-        basepath: /marathon/v2/reflect/me
-        endpoint_id: http://127.0.0.1:8080
-        redirect_testscases:
-          - location_expected: http://127.0.0.1/service/marathon/v2/reflect/me
-            location_set: http://127.0.0.1/marathon/v2/reflect/me
-          - location_expected: http://127.0.0.1/service/marathon/v2/reflect/me
-            location_set: http://127.0.0.1/v2/reflect/me
-          - location_expected: http://127.0.0.1/service/marathon/v2/reflect/me
-            location_set: /v2/reflect/me
+        locations:
+          - path: /marathon
+            code: 307
```

Comment:
Reading @jgehrcke's comment in https://jira.mesosphere.com/browse/DCOS_OSS-1163 it seems that we want to do `308`.